ledger-core/v2/go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/golang/protobuf v1.3.2
 	github.com/google/gofuzz v1.0.0
 	github.com/google/gops v0.3.6
-	github.com/google/uuid v1.1.1
+	github.com/google/uuid v1.1.1 // indirect
```

Comment:
run go mod tidy. I'm not sure why this dependency keep toggling, but it should be direct